IrConsoleApp/Lecture6/Person.cs

```diff
@@ -0,0 +1,60 @@
+﻿using IrynaZharnasek.Lecture6;
+
+namespace IrynaZharnasek.Lecture6
+{
+    public class Person
+    {
+        private int age;
+        //defining access metod for Student's Age:
```

Comment:
add empty line before comments (here and below)

Reply:
Done